src/components/CreateAccountForm/CreateAccountForm.js

```diff
@@ -165,13 +218,13 @@ export default class CreateAccountForm extends Component {
           <div className='form-group'>
             <label className='mr-1'>
               <p>Password</p>
-              <input type='password' className='form-control' required value={this.state.password} onChange={e => {
+              <input type='password' className='form-control' value={this.state.password} onChange={e => {
```

Comment:
I think the password and confirmPassword should be required for creating an account. `required={!this.state.edit}` should work